cmd/ctr/commands/tasks/start.go

```diff
@@ -31,28 +31,7 @@ var startCommand = cli.Command{
 	Name:      "start",
 	Usage:     "start a container that has been created",
 	ArgsUsage: "CONTAINER",
-	Flags: append(platformStartFlags, []cli.Flag{
-		cli.BoolFlag{
-			Name:  "null-io",
-			Usage: "send all IO to /dev/null",
-		},
-		cli.StringFlag{
-			Name:  "log-uri",
-			Usage: "log uri",
-		},
-		cli.StringFlag{
-			Name:  "fifo-dir",
-			Usage: "directory used for storing IO FIFOs",
-		},
-		cli.StringFlag{
```

Comment:
question : The pid-file, earlier it was present in both container flags as well as tasks flags right?  Now its moved only to task flags, doesnt the container section need that?

Reply:
Yes, it has no meaning when the container is created, only when the container is run(start task) the pid is recorded.